lib/common/session_model.dart

```diff
@@ -305,12 +294,16 @@ class SessionModel extends Model {
     );
   }
 
-  Future<void> setProxyAll<T>(bool on) async {
-    unawaited(
-      methodChannel.invokeMethod('setProxyAll', <String, dynamic>{
-        'on': on,
-      }),
-    );
+  Future<void> setProxyAll<T>(bool isOn) async {
+    if (isMobile()) {
```

Comment:
I think we need to remove Mobile call here, We don't have any method for proxyAll

Reply:
Will do!